docs/tutorials/chat-objects.md

```diff
@@ -0,0 +1,280 @@
+            "name": "All documents [0.00s]",
+            "status": "completed",
+            "attachments": [
+              {
+                "index": 0,
+                "type": "text/markdown",
+                "title": "[0] 'my_test_folder/test.docx'",
+                "data": "This is a test document",
+                "reference_url": "files/8kYT43q2Qt8ANLzV17Xq12128AFcKWpUkK7xoFPKXAB2dL1ztpMssPDBEEwLdvG3Rs/my_test_folder/test.docx"
+              }
+            ]
+          }
+        ]
+    }
+]
+```
+
+> Refer to [Completions API](https://epam-rail.com/dial_api#/paths/~1openai~1deployments~1%7BDeployment%20Name%7D~1chat~1completions/post) to view the description of all fields.
+
+Following the above pattern, you can render three stages in AI DIAL Chat UI:
+
+![](./img/att-stages.png)
+
+## Markdown
+
+AI DIAL Chat natively supports markdown. 
+
+You can create a message using a standard markdown notation:
+
+```json
+  "messages": [
+    {
+      "role": "user",
+      "content": "[link](api/files/bucket/your_image.jpg)\n\n![image](api/files/bucket/your_image.jpg)\n\n| Month    | Savings |\n| -------- | ------- |\n| January  | $250    |\n| February | $80     |\n| March    | $420    |\n\n",
+      ...
+    }
+  ]
+```
+
+This message will be rendered in AI DIAL Chat UI like this: 
+
+![](./img/markdown.png)
+
+
+## Visualizers
+
+We call Visualizers special applications that are used for rendering a specific type of content in AI DIAL Chat UI. AI DIAL Chat comes with built-in support for the [Plotly data visualizer](#plotly). You can use [DIAL Chat Visualizer Connector](https://github.com/epam/ai-dial-chat/blob/development/libs/chat-visualizer-connector/README.md) library to create visualizers for specific types of content.
+
+Watch a [demo video](../video%20demos/demos-for-developers/dial-data-viz) to view how a financial data can be visualized in AI DIAL Chat.
+
+## Plotly
+
+AI DIAL Chat has a built-in support for Plotly data visualization library. We use a specific MIME type for Plotly: 
+
+`"type": "application/vnd.plotly.v1+json"`
+
+To attach a Plotly visualization in the response of your application, follow the example below:
+
+```json
+  "messages": [
+    {
+      "custom_content": {
+        "attachments": [
+          {
+            "index": 1,
+            "type": "application/vnd.plotly.v1+json",
+            "title": "Your visualization name",
+            "url": "files/bucket/appdata/app_name/your_viz_file.json"
+          }
+        ]
+      }
+    }
+  ]
+```
+
+Similar structure in the application response will include a visualization as an attachment in AI DIAL Chat UI: 
+
+![](./img/att-plotly.png)
+
+
+Watch a [demo video](../video%20demos/demos/animated-scatterplot) to view how a statistical data can be visualized in AI DIAL Chat in Plotly animated scatterplot or a demo video of the [Omics AI Assistant](../video%20demos/demos/dial-omics-assistant) to see how protein structures can be visualized using Plotly.
```

Comment:
`../video%20demos/demos/dial-omics-assistant`

this link leads to a 404 page. Is it expected? Maybe need to check the other ones

Reply:
Will be fine on the website